package ExceptionHandling;

public class Main 
{
	public static void main(String[] args) 
	{
		int c = 0;
		try
		{
			
		}
		catch(ArithmeticException e)
		{
			
		}
		catch(Exception e)
		{
			
		}
	}
}
